src/RecordComponent.cpp

```diff
@@ -74,6 +77,25 @@ RecordComponent::getExtent() const
     return m_dataset->extent;
 }
 
+RecordComponent&
+RecordComponent::makeEmpty( Dataset d )
+{
+    if( written )
+        throw std::runtime_error(
+            "A recordComponent can not (yet) be made"
```

Comment:
`RecordComponent` is a type, so let's keep the spelling to the typename as above:
```suggestion
            "A RecordComponent cannot (yet) be made"
```